src/app-bundles/corporate-office-bundle.js

```diff
@@ -14,25 +13,21 @@ export default createRestBundle({
   putTemplate: null,
   postTemplate: null,
   deleteTemplate: null,
-  activeRoutes: [ RoutePaths.CorpOfficeList ],
-  fetchActions: ["APP_INITIALIZED","URL_UPDATED"],
```

Comment:
Without the a fetch action defined, what triggers this bundle to load it's data?

Reply:
The fetch is triggered directly by the CorpOfficeReportsPage in the useEffect. This way it's only called when the corporate reports page is rendered